.github/workflows/test.yml

```diff
@@ -69,15 +73,20 @@ jobs:
           echo TF_ACC_PROVIDER_NAMESPACE="hashicorp" >> $GITHUB_ENV
           echo TF_ACC_PROVIDER_HOST="registry.opentofu.org" >> $GITHUB_ENV
 
-      - run: make testacc TESTARGS="-coverprofile=coverage.txt -coverpkg=./..."
+      - run: go test -v -timeout=30m -parallel=8 -coverprofile=coverage.txt -coverpkg=./... ./...
         env:
           # Domain must be available in the account running the tests. This domain is
           # available in the account running the public integration tests.
           CERT_DOMAIN: hc-integrations-test.de
 
+          TF_ACC: 1
           TF_LOG: DEBUG
           TF_LOG_PATH_MASK: test-%s.log
 
+      - run: |
+          mv coverage.txt coverage.txt.tmp
+          grep -v "/test" coverage.txt.tmp > coverage.txt
```

Comment:
Alternatively codecov could be configured to ignore these paths, i dont mind either way: https://docs.codecov.com/docs/codecovyml-reference#ignore

---

Could you add a comment to explain why these paths are not considered for coverage?

---
Right now codecov still shows coverage for `internal/teste2e/testing.go`: https://app.codecov.io/gh/hetznercloud/terraform-provider-hcloud/pull/850/blob/internal/teste2e/testing.go (as well as `testsupport` & `testtemplate`)